### PR TITLE
[8.x.x] Form layout manager new split-pane mode and refactor

### PR DESCRIPTION
**Pull request by gonzalo-martinez-imt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
     "reflect-metadata": "0.1.13",
     "rxjs": "~6.4.0",
     "tslib": "~1.10.0",
-    "zone.js": "0.9.1"
+    "zone.js": "0.9.1",
+    "angular-resizable-element": "3.3.4"
```
Comment (daniel-grana) [inline package.json:37]: I think that it is useful to maintain the libraries odered alfabetically

File: projects/ontimize-web-ngx/ng-package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
       "@ngx-translate/core": "@ngx-translate/core",
       "@ngx-translate/http-loader": "@ngx-translate/http-loader",
       "moment": "moment",
-      "ngx-material-timepicker": "ngx-material-timepicker"
+      "ngx-material-timepicker": "ngx-material-timepicker",
+      "angular-resizable-element": "angularResizableElement"
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/ng-package.json:12]: The same as previous commment

File: projects/ontimize-web-ngx/ng-package.json [hunk 3]
```diff
@@ -17,6 +18,7 @@
     "moment",
     "ngx-material-timepicker",
     "reflect-metadata",
-    "tslib"
+    "tslib",
+    "angular-resizable-element"
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/ng-package.json:22]: The same as previous commment

File: projects/ontimize-web-ngx/package.json [hunk 4]
```diff
@@ -56,6 +56,7 @@
     "@ngx-translate/http-loader": "~4.0.0",
     "moment": "2.24.0",
     "ngx-material-timepicker": "5.4.3",
-    "reflect-metadata": "0.1.13"
+    "reflect-metadata": "0.1.13",
+    "angular-resizable-element": "3.3.4"
```
Comment (daniel-grana) [inline projects/ontimize-web-ngx/package.json:60]: The same as previous commment

**Review by daniel-grana — CHANGES_REQUESTED** [review]

File: projects/ontimize-web-ngx/src/lib/layouts/form-layout/split-pane/o-form-layout-split-pane.component.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<div fxFill>
```
Comment (manuel-cid) [inline projects/ontimize-web-ngx/src/lib/layouts/form-layout/split-pane/o-form-layout-split-pane.component.html:1]: Is this `div` needed? Can we move the `fxFill` to the next `div` that already has `height: 100%` in the class `row-wrapper`

File: projects/ontimize-web-ngx/src/lib/layouts/form-layout/split-pane/o-form-layout-split-pane.component.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    <div #detailWrapper class="content-wrapper" mwlResizable [enableGhostResize]="true"
+      (resizeEnd)="onResizeEnd('width', $event, detailWrapper)">
+
+      <div class="resize-handle" mwlResizeHandle [resizeEdges]="{ left: true }">
+        <div class="resize-border">
+          <div></div>
+        </div>
+      </div>
+
+      <div class="secondary-content" fxLayout="column" fxLayoutAlign="center stretch" fxFlex>
```
Comment (manuel-cid) [inline projects/ontimize-web-ngx/src/lib/layouts/form-layout/split-pane/o-form-layout-split-pane.component.html:16]: Can we rename css classes to something that starts with `o-` and makes reference to the current component?

**Review by manuel-cid — CHANGES_REQUESTED** [review]

File: projects/ontimize-web-ngx/src/lib/components/form/navigation/o-form-navigation.component.ts [hunk 7]
```diff
@@ -256,8 +256,7 @@ export class OFormNavigationComponent implements OnDestroy {
     }
   }
 
-  private moveInDialogManager(formLayoutManager: any, index: number) {
-    formLayoutManager.dialogRef.componentInstance.urlParams = this.navigationData[index];
+  private moveInFormLayoutManager(formLayoutManager: OFormLayoutManagerComponent, index: number) {
```
[the remark this answers is not in the document]
Comment (gonzalo-martinez-imt) [reply projects/ontimize-web-ngx/src/lib/components/form/navigation/o-form-navigation.component.ts:259]: You are right, besides of that we would also have the reference to the formLayoutManager in the compnent using this.formLayoutManager. Thank you

**Review by manuel-cid — DISMISSED** [review]

**Review by daniel-grana — APPROVED** [review]

**Review by manuel-cid — APPROVED** [review]

**Review by pacogomezimatia — APPROVED** [review]

**Pull request merged**